dev/com.ibm.ws.security.social/src/com/ibm/ws/security/social/tai/SocialLoginWithAccessToken.java

```diff
@@ -0,0 +1,64 @@
+package com.ibm.ws.security.social.tai;
+
+import java.util.Map;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import org.osgi.service.component.ComponentContext;
+import org.osgi.service.component.annotations.Activate;
+import org.osgi.service.component.annotations.Deactivate;
+import org.osgi.service.component.annotations.Modified;
+
+import com.ibm.websphere.ras.Tr;
+import com.ibm.websphere.ras.TraceComponent;
+import com.ibm.websphere.security.WebTrustAssociationException;
+import com.ibm.websphere.security.WebTrustAssociationFailedException;
+import com.ibm.ws.security.social.Constants;
+import com.ibm.ws.security.social.TraceConstants;
+import com.ibm.ws.security.social.internal.utils.SocialTaiRequest;
+import com.ibm.wsspi.security.tai.TAIResult;
+
```

Comment:
Let's add a comment - The purpose of this class is to run before we look at ltpa cookie so that in event of looking for access token in header, we get a chance to look for it before we handle ltpa cookie flow